client/src/preview/components/asset/AssetBoard.tsx

```diff
@@ -34,21 +36,35 @@ const AssetGridItem: React.FC<{
     lg={3}
     sx={{ minWidth: 250 }}
   >
-    <AssetCardExecute asset={asset} />
+    {tab === 'Execute' ? (
+      <AssetCardExecute asset={asset} />
+    ) : (
+      <AssetCardManage asset={asset} onDelete={() => onDelete(asset.path)} />
```

Comment:
The `onDelete` function is passed exclusively to `AssetCardManage`, which includes the Delete button. This is because it is the only button that, in addition to having an API call via `gitlabDigitalTwin`, must also modify the store via dispatch with the `deleteAsset` function.